torch/onnx/ops/__init__.py

```diff
@@ -0,0 +1,224 @@
+from __future__ import annotations
```

Comment:
I think this folder/files worth to be specified as something like "torch.fx related XXXX", and we should list out the pieces we need to register it. It will be helpful to bridge more robust relationship between onnx and torch.